client/components/ItemName.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+
+
+
+class ItemName extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+
+    }
+  }
+
+  render() {
+    return (
+      <div style={{fontSize: 25}}>
```

Comment:
Is this styling rendering as you expect? I am unfamiliar with this syntax.

Reply:
Yes. This is working for now. This is inline styling in react.

Reply:
Gotcha. Hadn't gotten around to that yet with my own module.